runscope/resource_runscope_test_.go

```diff
@@ -37,19 +37,27 @@ func resourceRunscopeTest() *schema.Resource {
 			},
 			"default_environment_id": &schema.Schema{
 				Type:     schema.TypeString,
-				Computed: true,
+				Computed: false,
```

Comment:
Build failing with "one of optional, required, or computed must be set"
```suggestion
				Optional: true,
```